### DISCUSSION TITLE
Where and how are you using Photino?

### DISCUSSION THREAD

**Discussion by ottodobretsberger** [tryphotino/photino.NET#172] [Polls]
In order to better prioritize features and development, we'd like to ask where and how the active community is using Photino. Please take a minute for this poll, we would greatly appreciate it!

**Comment by Xyncgas** [comment]
I simply prefer web UI than other .NET UI framework
Using photino I integrated existing web UI library in .NET to deliver content to people
Photino essentially offered a separate environment for me to do computation and IO without browser restrictions for example CORS are annoying, me negotiating domain boundary in my application enable freedom for me a developer to do stuff however I want to, enabling a product delivery better than using Xamarin-liked technology

**Reply by Xyncgas** [reply]
WebUI works because browsers have the details polished for rendering everything already, and I am seeing it the most sensible way as putting web front-end to whatever program needing a GUI, the convenience for reusable code for maximum platform reach is valuable and we are worthy of having the potentials for a .NET application generally available to have Web GUI on every platform it's running that can run a browser

**Comment by raddevus** [comment]
I'm ramping up usage of Photino & hoping a couple of issues will be resolved:
1) installers - without good packaging people will never feel safe enough to install apps.
   My apps will be cross-platform (developing on Ubuntu)
    a) I will want to generate a .deb, .rpm, snap, etc.
    b) I will want to generate dmg for mac silicon macOS (i have a mac pro (M4) that i dev on)
    c) I will want to be able to generate proper packaging for win app store 
         -- I've built my app C'YaPass in Electron & deployed it to the windows store so I'll want to do that with photino apps too
  2) Got to have the icon loading stuff worked out or the app will never look like a valid app to users.
 I'm building out a dedicated site (allos.dev -- nothing there yet) to cross-platform development & want photino to be the basis of the apps, without these two things it won't go.  
 
## Big Question
 How much do the photino team dedicate to these forums?  I see a lot of unaswered questions.  
 I'm happy to lend a hand where I can.

**Reply by MikeYeager** [reply]
Packaging is a bit out of our scope, but we do have some pretty good documentation now, thanks to @philippjbauer. Please see [packaging guide in samples](https://github.com/tryphotino/photino.Samples/blob/master/Photino.PublishPhotino/PublishPhotino/README.md). It also discusses application icons there.

**Comment by Cole-Jaeger** [comment]
We are developing an enterprise web app to replace a .net framework legacy standalone app. We are developing this front interface using angular, with a .net api back end. 

What we are trying to accomplish is to take this web infrastructure and host it in a local app. 

Looks like you support hosting a web api with a web interface on startup.

We are rather early in development of the front-end assets, and I am looking into what we are going to use to accomplish our goals and found photino. I am going to be creating a test bed to see how this package works and if this works for us and just posted here since you were asking how people are looking to use Photino. 

From what I'm reading seems like this will work for us but curious on anyone else's progress.

**Comment by Ari24-cb24** [comment]
I'm using Photino for a small commcerical project for athletic shooting equipment since it allows me to communicate to the driver via .NET and working with a clean web-frontend using vue. We might also use Photino for an upcoming school project but not too sure yet.

What I would really benefit is a Java backend since lots of our internal libraries are based on Java. Might look into developing one one day but can't say for sure.

**Reply by Xyncgas** [reply]
You can already use java and C# bridges today

**Comment by schaveyt** [comment]
I'm using for both personal and internal, line-of-business applications in conjunction with [Flowbite Blazor](https://flowbite-blazor.peakflames.org/)

**Comment by Dwarf1er** [comment]
Currently using it for personal projects, but started testing possibly creating internal tooling for a small team.

**Comment by 0use-TE** [comment]
Currently using it for personal projects,for example,build some tools.

**Comment by Sorrow-Scarlet** [comment]
Evaluating photino for lightweight AOT F# hobby projects...
